crates/plugin/src/plugin.rs

```diff
@@ -399,24 +411,37 @@ impl GeyserPlugin for GeyserPluginRabbitMq {
 
                 let stx: &SanitizedTransaction;
                 let meta: &TransactionStatusMeta;
+                let is_vote: bool;
 
                 match transaction {
                     ReplicaTransactionInfoVersions::V0_0_1(tx) => {
-                        if matches!(tx.transaction_status_meta.status, Err(..)) || tx.is_vote {
-                            return Ok(());
-                        }
                         stx = tx.transaction;
                         meta = tx.transaction_status_meta;
+                        is_vote = tx.is_vote;
                     },
                     ReplicaTransactionInfoVersions::V0_0_2(tx) => {
-                        if matches!(tx.transaction_status_meta.status, Err(..)) || tx.is_vote {
-                            return Ok(());
-                        }
                         stx = tx.transaction;
                         meta = tx.transaction_status_meta;
+                        is_vote = tx.is_vote;
```

Comment:
If a new `ReplicaTransactionInfoVersions` were to release, v3, without a code change here how might that behave assuming `stx`, `meta`, and `is_vote` don't get initialized?

Reply:
This code would not compile anymore (all branches not handled), and would panic if attempted to be used against a version of Solana that had that change.

When that is added (there is a V3 in 1.16.x), when we compile for that version of Solana it'll tell us that needs handled.